pkg/server/logging.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2025 The Sigstore Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/logging"
+)
+
+func interceptorLogger(l *slog.Logger) logging.Logger {
```

Comment:
just wondering, is this needed so that logging is context-aware?

Reply:
or is this about changing the default?

Reply:
If we want log messages at the start/finish of each grpc method, you have to connect the go-grpc-middleware to the slog Logger. right now that's disabled in L30 of this file, but if we fix the TODO here we'll need that.

Reply:
Sounds good, can you file an issue to track that?